packages/mdc-drawer/index.js

```diff
@@ -74,14 +65,15 @@ export class MDCDrawer extends MDCComponent {
   }
 
   destroy() {
-    document.removeEventListener('keydown', this.handleKeydown_);
+    this.root_.removeEventListener('keydown', this.handleKeydown_);
```

Comment:
Why was this changed to the root element? It doesn't seem to close when I his escape.

Reply:
Wanted to make ESC work only when the focus is on Drawer. So this works only when the focus is on Drawer. Hope that makes sense.

Reply:
I did click into the drawer, and hit escape, but it didn't close. Are you seeing this behavior as well?

Reply:
keydown wouldn't trigger when drawer is not in focus. drawer itself is not focusable element but nav items are.